gitgud/skills/parsing.py

```diff
@@ -1,6 +1,20 @@
 from copy import deepcopy
 
 
+def parse_solution(solution_path):
+    solution_commands = []
+
+    if not solution_path:
+        return solution_commands
```

Comment:
What's this for? When is `solution_path` going to be `None`?

Reply:
I implemented this for the case where `solutions.txt` doesn't exist as a file (because the level doesn't need solutions at all). The intro level, for example, doesn't have any solutions because the content of `test.spec` is the same as `setup.spec`.

Should we just operate under the assumption that all levels will have a `solution.txt` file? @benthayer

Reply:
I've just implemented it so that gitgud assumes there's always a `solutions.txt`.